rulesengine/rules/models.py

```diff
@@ -137,6 +144,7 @@ def populate(self, values):
         self.neg_surt = values.get("neg_surt", "")
         self.protocol = values.get("protocol", "")
         self.subdomain = values.get("subdomain", "")
+        self.status_code = values.get("status_code")
```

Comment:
Do we want a default here, too?

Or maybe not, since this field can be blank or null?